documentation.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page_md
 title: Documentation
-tagline: A comprehensive list of links to each of the repositories for the Jupyter project.
+tagline: A comprehensive list of Jupyter subprojects
```

Comment:
```suggestion
tagline: A comprehensive list of Jupyter projects and subprojects repositories
```

"subproject" has a very specific meaning https://jupyter.org/governance/newsubprojects.html but this list includes respoitories which are not a standalone subprojects - just standalone different pieces of software